packages/common/src/downloads/_internal/file-url-fetchers/fetchHubApiDownloadFile.ts

```diff
@@ -97,44 +114,77 @@ function getDownloadApiRequestUrl(options: IFetchDownloadFileOptions) {
  * Polls the Hub Download API until the download is ready, then returns the download file URL
  *
  * @param requestUrl Hub Download Api URL to poll
+ * @param pollInterval interval in milliseconds to poll for job completion
+ * @param cacheQueryParam an optional query param to control cache behavior. 'updateCache' should
+ * only be used with the initial request when a download cache update is desired. All subsequent
+ * requests should use'trackCacheUpdate' to follow the progress of the cache update.
+ * for subsequent requests when polling the progress of a cache update.
  * @param progressCallback an optional callback to report download generation progress
  * @returns the final file URL
  */
 async function pollDownloadApi(
   requestUrl: string,
   pollInterval: number,
+  cacheQueryParam?: CacheQueryParam,
   progressCallback?: downloadProgressCallback
 ): Promise<IFetchDownloadFileResponse> {
-  const response = await fetch(requestUrl);
+  // If requested, append the cache query param to the request URL
+  let withCacheQueryParam = requestUrl;
+  if (cacheQueryParam) {
+    withCacheQueryParam = `${requestUrl}&${cacheQueryParam}=true`;
```

Comment:
Are we assuming that existing query params are present?  I'm just wondering `?` vs `&` is going to cause an issue, eg:

`${requestUrl}?${cacheQueryParam}=true` vs `${requestUrl}&${cacheQueryParam}=true`

Reply:
Good question @tannerjt! As of now, the Download API requires a `?layers` query param in order to function:

`/v1/:itemId/:format?layers=<:layers>`

That's why we can reliably just add &.